sdk/cognitivelanguage/Azure.AI.Language.Conversations/src/Generated/ConversationAnalysisClient.cs

```diff
@@ -388,6 +392,8 @@ public virtual Response AnalyzeConversation(RequestContent content, RequestConte
         /// Get the status of an analysis job. A job may consist of one or more tasks. Once all tasks are succeeded, the job will transition to the succeeded state and results will be available for each task.
         /// 
         /// Below is the JSON schema for the response payload.
+        /// Additional information can be found in the service REST API documentation:
+        /// https://docs.microsoft.com/rest/api/language/conversation-analysis-runtime/ob-status
```

Comment:
404s

Reply:
A bunch below will as well. Looks like the last path component here and below is missing a few characters at the beginning.

Reply:
This value is from https://github.com/Azure/azure-sdk-for-net/blob/135dd405d0c0792b90364a06602235bb8e9edc76/sdk/cognitivelanguage/Azure.AI.Language.Conversations/src/autorest.md?plain=1#L160-L176

Would you update the directives to valid version?

Reply:
Oh, so this change only enabled `externalDocs` to work and didn't add them itself? That wasn't obvious from the PR title or description. If that's the case, I will approve this and open a separate but to fix that issue.

Reply:
> so this change only enabled externalDocs to work

Correct. We upgrade the M4 version which enables the `externalDocs` feature.